modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,53 @@
+[id="logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+.Procedure
+
+. After creating the Argo CD instance, login into the keycloak console.
```

Comment:
Since all your other procedure steps begin with an action verb, I would use that same sentence construction here so that you are consistent. 

So consider rephrasing to something like, 
`Log into the Keycloak Console after creating the Argo CD instance: <snippet>.`

Reply:
Note, use `log into` when it's a verb/action the user takes. 

Use `login` when it's a noun. Like, `Select the `login` button.`